Source/Core/GeometryPipeline.js

```diff
@@ -1986,6 +2006,26 @@ define([
             Cartesian3.pack(direction, currentAttributes.extrudeDirection.values, insertedIndex * 3);
         }
 
+        if (defined(applyOffsets)) {
```

Comment:
Doesn't #6644 handle this?

Reply:
No, since it's a boolean property we don't want to interpolate it.  The value should always be `1.0` or `0.0`.  It took me forever to figure out the correct values here, I really don't think it can be generalized.

Reply:
Why can't you interpolate it? If all vertices have a `0.0`, the interpolation will be `0.0`. If all vertices have `1.0`, the interpolation will be `1.0`, If the interpolation is between `0.0` and `1.0`, it should be `1.0` so set it to `Math.ceil(interpolatedValue)`.

Reply:
Okay, I figured it out.  It was just an epsilon problem that was causing the barycentric interpolation to go wrong